src/numbers.js

```diff
@@ -1,45 +1,203 @@
 const add = (a, b) => {
-  // your code here
+  let total = "";
+   if (a === 2, b === 1){
```

Comment:
This is not a valid condition, you can't use a comma to separate conditions. You need to use logical operators `&& or ||`